### PR TITLE
Add documentation for cakeissues.net

### PR DESCRIPTION
**Pull request by pascalberger**
Add documentation for cakeissues.net, moved from Cake.Issues.Website repository including full history.

Part of https://github.com/cake-contrib/Cake.Issues.Website/issues/123

### CODE REVIEW

File: docs/examples.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    // Read Issues.
+    var issues = ReadIssues(
+        DocFxIssuesFromFilePath(logPath, docRootPath),
+        repoRootPath);
+
+    Information("{0} issues are found.", issues.Count());
+});
+```
+
+[DocFx]: http://dotnet.github.io/docfx/
```
Comment (x-jokay) [inline docs/examples.md]: Use `https` for link.

File: docs/examples.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    });
+});
+```
+
+Finally you can define a task where you call the core addin with the desired issue provider.
+
+```csharp
+Task("Analyze-Log")
+.IsDependentOn("Build-Documentation")
+.Does(() =>
```
Comment (x-jokay) [inline docs/examples.md]: Fix indention for `.IsDependentOn` and `.Does`.

File: docs/examples.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    var issues = ReadIssues(
+        DocFxIssuesFromFilePath(logPath, docRootPath),
+        repoRootPath);
+
+    Information("{0} issues are found.", issues.Count());
+});
+```
+
+[DocFx]: http://dotnet.github.io/docfx/
+[Cake.DocFx]: https://www.nuget.org/packages/Cake.DocFx/
```
Comment (x-jokay) [inline docs/examples.md]: [MD047](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md047---files-should-end-with-a-single-newline-character) markdown files should end with a single newline character.

File: docs/features.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.Message`                  |                                |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.Priority`                 | Always [IssuePriority.Warning] |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.PriorityName`             | Always [IssuePriority.Warning] |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.Rule`                     |                                |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.RuleUrl`                  |                                |
+
+[Cake.Issues.DocFx addin]: https://www.nuget.org/packages/Cake.Issues.DocFx
+[DocFx]: https://dotnet.github.io/docfx/
+[Cake.DocFx]: https://www.nuget.org/packages/Cake.DocFx
+[IssuePriority.Warning]: ../../../api/Cake.Issues/IssuePriority/7A0CE07F
```
Comment (x-jokay) [inline docs/features.md]: [MD047](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md047---files-should-end-with-a-single-newline-character) markdown files should end with a single newline character.

File: docs/requirements.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+---
+Order: 10
+Title: Requirements
+Description: Requirements for the Cake.Issues.DocFx addin.
+---
+The requirements for using the [Cake.Issues.DocFx addin] are listed in the [release notes] for any specific version.
+
+[Cake.Issues.DocFx addin]: https://www.nuget.org/packages/Cake.Issues.DocFx
+[release notes]: release-notes
```
Comment (x-jokay) [inline docs/requirements.md]: [MD047](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md047---files-should-end-with-a-single-newline-character) markdown files should end with a single newline character.

File: docs/features.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.ProviderName`             |                                |
+| <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IIssue.ProjectName`              |                                |
+| <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IIssue.ProjectFileRelativePath`  |                                |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.AffectedFileRelativePath` |                                |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.Line`                     |                                |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.Message`                  |                                |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.Priority`                 | Always [IssuePriority.Warning] |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.PriorityName`             | Always [IssuePriority.Warning] |
```
Comment (x-jokay) [inline docs/features.md]: `Always [IssuePriority.Warning]` seems not correct for `IIssue.PriorityName`.

File: docs/features.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+# Basic features
+
+* Reads warnings from [DocFx] log files.
+
+  :::{.alert .alert-info}
+  [DocFx] can be run with [Cake.DocFx] addin.
+  :::
+
+# Supported comment formats
```
Comment (x-jokay) [inline docs/features.md]: [MD025](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md025---multiple-top-level-headings-in-the-same-document
) multiple top level headings in the same document.

File: docs/features.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+|                                                                    | Comment format                 | Remarks                        |
+|--------------------------------------------------------------------|--------------------------------|--------------------------------|
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IssueCommentFormat.PlainText` |                                |
+| <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IssueCommentFormat.Markdown`  |                                |
+| <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IssueCommentFormat.Html`      |                                |
+
+# Supported IIssue properties
```
Comment (x-jokay) [inline docs/features.md]: [MD025](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md025---multiple-top-level-headings-in-the-same-document
) multiple top level headings in the same document.

File: docs/features.md (hunk 6 above)
Comment (pascalberger) [reply docs/features.md]: You're right in the meantime there could also be suggestions

**Comment by pascalberger** [comment]
@x-jokay Ready for another review.

Top level headings are  required to have the entry appear in site TOC (See https://github.com/cake-contrib/Cake.Issues.Website/pull/41)

**Pull request merged**